translations/ui-plugin-bursar-export/en.json

```diff
@@ -1,5 +1,6 @@
 {
   "meta.title": "Transfer configuration",
+  "meta.settingsTitle": "Users settings - Transfer criteria",
```

Comment:
Meta is special for the settings pane title only - use something else here (title.settings or something)

Doesn't matter as much for this, but still probably would be good